build.gradle

```diff
@@ -1,6 +1,6 @@
 plugins {
     id "java"
-    id "edu.wpi.first.GradleRIO" version "2024.1.1"
+    id "edu.wpi.first.GradleRIO" version "2024.2.1"
```

Comment:
Didn't know we weren't up to date -- we should make sure this particular change goes in.